mainapp/models.py

```diff
@@ -272,6 +272,11 @@ class RescueCamp(models.Model):
     taluk = models.CharField(max_length=50,verbose_name="Taluk - താലൂക്ക്")
     village = models.CharField(max_length=50,verbose_name="Village - വില്ലജ്")
     contacts = models.TextField(verbose_name="Phone Numbers - ഫോൺ നമ്പറുകൾ",blank=True,null=True)
+    facilities_available = models.TextField(
+        blank=True,
+        null=True,
```

Comment:
is null=True required?

Reply:
Ideally, it should not be set on Textfields, still adding as PR is urgent

Reply:
Okay. The column was already there, I've only rearranged.